models/stg_amplitude__event.sql

```diff
@@ -19,7 +19,7 @@ fields as (
 
 final as (
     
-    select 
+    select  
```

Comment:
What is the point of the `final_pre` and the `final` ctes in this model? They seem like they can be combined into one. Thoughts?

Reply:
Yeah I did my column renaming in the `final_pre` (`id as event_id`) before referencing it in the surrogate key in `final`, to keep it clean, but it could go either way!

Reply:
cleaned up to just one cte!